### PR TITLE
Add Telemetry for coreclr-debug acquisition

### PR DESCRIPTION
**Pull request by chuckries**
Wish I would have gotten this in yesterday... I'd like to potentially push out another version of the extension this week so we can get this in.

### CODE REVIEW

**Comment by chuckries** [comment]
@gregg-miskelly @DustinCampbell

File: src/coreclr-debug.ts [hunk 1]
```diff
@@ -24,6 +26,8 @@ export function installCoreClrDebug(context: vscode.ExtensionContext) {
         return;
     }
     
+    initializeTelemetry(context);
```
Comment (gregg-miskelly) [inline src/coreclr-debug.ts]: Might want to move this to line 36 for now so that we do as little as possible when .NET CLI isn't installed.

**Comment by gregg-miskelly** [comment]
:+1:

**Comment by DustinCampbell** [comment]
LGTM

**Comment by DustinCampbell** [comment]
go ahead and merged. I tagged our release in master already.

File: src/coreclr-debug.ts (hunk 1 above)
Comment (chuckries) [inline src/coreclr-debug.ts]: done.

**Pull request merged**